paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -270,14 +265,14 @@ internal class DefaultFlowController @Inject internal constructor(
             else -> null
         }?.let { confirmParams ->
             lifecycleScope.launch {
-                paymentController.startConfirmAndAuth(
-                    authHostSupplier(),
-                    confirmParams,
-                    ApiRequest.Options(
-                        apiKey = lazyPaymentConfiguration.get().publishableKey,
-                        stripeAccount = lazyPaymentConfiguration.get().stripeAccountId
-                    )
-                )
+                when (confirmParams) {
+                    is ConfirmPaymentIntentParams -> {
+                        paymentLauncher?.confirm(confirmParams)
```

Comment:
There was a confirm and startCOnfirmAndAuth methods.   Are these now combined?

Reply:
Yes! That's part of the reason PaymentLauncher is so much better, it handled that intermediate step.